<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Display_Window_Ui</class>
 <widget class="QWidget" name="Display_Window_Ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1244</width>
    <height>1074</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TurnFix</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/icon.png</normaloff>:/icons/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string>background-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="main">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../../resources/TurnFix.qrc">:/icons/turnfix-gross.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout">
       <property name="verticalSpacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item row="8" column="0">
        <widget class="QWidget" name="wdg_score" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_last">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 0);
font-size:100px;
font-family:Arial Narrow;
font-weight:bold;</string>
            </property>
            <property name="text">
             <string>test</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_score">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 0);
font-size:175px;
font-family:Arial;
font-weight:bold;</string>
            </property>
            <property name="text">
             <string>0.000</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="wdg_head" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>217</width>
              <height>147</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_icon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Tahoma</family>
             </font>
            </property>
            <property name="styleSheet">
             <string>background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="pixmap">
             <pixmap resource="../../../resources/TF_GeraeteIcons.qrc">:/icons/reck.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="lbl_time">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 10, 154);
font-size:130px;
font-family:Arial;
font-weight:bold;</string>
            </property>
            <property name="text">
             <string>0:28</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QStackedWidget" name="sta_ampel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page_3">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="lbl_green">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>150</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>150</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../../../resources/TurnFix.qrc">:/andere/ampel_gruen.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_4">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="lbl_red">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>150</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>150</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../../../resources/TurnFix.qrc">:/andere/ampel_rot.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>147</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lbl_geraet">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 10, 154);
font-size:100px;
font-family:Arial;
font-weight:bold;
padding:1px;</string>
            </property>
            <property name="text">
             <string>RECK</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QWidget" name="wdg_current" native="true">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_current_nachname">
            <property name="font">
             <font>
              <family>Arial Narrow</family>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font-size:135px;
font-family:Arial Narrow;
font-weight:bold;</string>
            </property>
            <property name="text">
             <string>KRÄMER</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_current_vorname">
            <property name="font">
             <font>
              <family>Arial Narrow</family>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font-size:115px;
font-family:Arial Narrow;
font-weight:bold;</string>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lbl_current_club">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 248, 0);
font-size:70px;
font-family:Arial Narrow;</string>
            </property>
            <property name="text">
             <string>SKV NAUHEIM Ä</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="lbl_current_snr">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 248, 0);
font-size:70px;
font-family:Arial Narrow;</string>
            </property>
            <property name="text">
             <string>1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="Line" name="line">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>10</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>10</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>15</number>
         </property>
         <property name="midLineWidth">
          <number>15</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="Line" name="line_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>5</number>
         </property>
         <property name="midLineWidth">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="Line" name="line_3">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>5</number>
         </property>
         <property name="midLineWidth">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lbl_next">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font-size:90px;
font-family:Arial Narrow;
font-weight:bold;</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_5">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="5" column="0" colspan="3">
        <widget class="QLabel" name="lbl_verein">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 221, 0);
font-size:130px;
font-family:Arial Narrow;
font-weight:bold;</string>
         </property>
         <property name="text">
          <string>TG Rüsselsheim</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="3">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_icon_2">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../../resources/TF_GeraeteIcons.qrc">:/icons/sprung.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLabel" name="lbl_einturnen">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 10, 154);
font-size:130px;
font-family:Arial;
font-weight:bold;</string>
         </property>
         <property name="text">
          <string>EINTURNEN</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>268</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QStackedWidget" name="sta_ampel_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="page_6">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="lbl_green_2">
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../../../resources/TurnFix.qrc">:/andere/ampel_gruen.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_7">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="lbl_red_2">
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../../../resources/TurnFix.qrc">:/andere/ampel_rot.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_time_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 10, 154);
font-size:130px;
font-family:Arial;
font-weight:bold;</string>
            </property>
            <property name="text">
             <string>0:00</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>268</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_8">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_zwischen">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 10, 154);
font-size:130px;
font-family:Arial;
font-weight:bold;</string>
         </property>
         <property name="text">
          <string>Zwischenstand</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tbl_overview">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white;
font-size:20px;</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>5</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/TF_GeraeteIcons.qrc"/>
  <include location="../../../resources/TurnFix.qrc"/>
 </resources>
 <connections/>
</ui>
